Sources/EngineKit/HashableExtensions.swift

```diff
@@ -46,118 +52,173 @@ extension Instructions: Hashable {
 // MARK: - Instruction + Hashable
 extension Instruction: Hashable {
 	public func hash(into hasher: inout Hasher) {
+		switch self {
```

Comment:
We already do this switch below, so if new cases are added it seems that the other switch will catch that, and the developer will be prompted to handle the new case in the place where that is actually needed. So what is the purpose of the extra switch?

Reply:
right, not needed for `hash:into`, only for equality, copy paste bug...